.github/ISSUE_TEMPLATE/documentation.yml

```diff
@@ -1,36 +1,35 @@
 name: Documentation request
 description: Raise an issue related to content.
-title: "📝 Doc:"
-labels: "documentation, to sort"
+title: "📝 Doc: "
+labels: ["documentation", "to sort"]
 body:
   - type: markdown
     attributes:
       value: |
-        * Please check that there is not an existing open [issue](https://github.com/tracim/tracim/labels/documentation) before you create a new one.
+        Thanks for wanting to improve Tracim!
+        Before you submit, please make sure the documentation change hasn't already been suggested by searching [open](https://github.com/tracim/tracim/issues) and [closed](https://github.com/tracim/tracim/issues?q=is%3Aissue+is%3Aclosed).
   - type: input
+    id: contents
     attributes:
-      label: What content on github.com/tracim/tracim is affected?
-      placeholder: "github.com/tracim/tracim"
+      label: What content on Tracim is affected?
+      placeholder: Tracim README.md, tracim_frontend_lib BUILD.md, etc...
```

Comment:
theses examples seems to be files rather than "contents"
I'd say to either say "which files on tracim is affected" of put "file, note, thread, kanban..." as placeholder